### ISSUE TITLE
Embedded projects no longer work

### ISSUE THREAD

**Issue by JetStarBlues** [Khan/live-editor#633]
Embedded projects (via share link) don't seem to be working at the moment. An alert pops up with the message:

> **The page at https://www.khanacademy.org says:**
> We couldn't load the program, can you try reloading?

Opening the console shows no error messages...

**Comment by kevinbarabash** [comment]
@JetStarBlues the embedded programs on https://www.khanacademy.org/computing/computer-programming/programming-games-visualizations/programming-transformations/a/translation are working.  Can you provide a link to a page with an embedded project that isn't loading?

**Comment by JetStarBlues** [comment]
@kevinbarabash I came across the problem on my blog. Maybe it's because it is not under the khanacademy.org domain? Here's a [link][0].

[0]: http://www.jk-quantized.com/blog/projects/khanAcademy/khanPosts/42_Asteroids.html

**Comment by JetStarBlues** [comment]
@kevinbarabash I tried opening the embeds I was referring to on my phone, and they worked. I also tried opening on a second computer and they worked there too... So, I think the problem is definitely with my computer.

(I assumed that testing on both Chrome and Firefox (both incognito and normal mode) was enough because it was a website but clearly it's important to check performance on other devices/computers).

Closing the issue. Thanks for your time.
